course_discovery/apps/course_metadata/data_loaders/tests/mock_data.py

```diff
@@ -3070,3 +3070,22 @@
     'uuid': '6b8b779f-f567-4e98-aa41-a265d6fa073a',
     'vuuid': 'e0f8c80a-b377-4546-b247-1c94ab3a218a'
 }
+
+WORDPRESS_API_BODIES = [
+    {
+        'course_id': 'course-v1:green+gn101+gn101',
```

Comment:
Always try to use course id's like `course-v1:edX+DemoX+Demo_Course`

```python
'course_id': 'course-v1:edX+DemoX+Demo_Course',
'title': 'Demo Course',
'slug': 'demo-course',
permalink': 'https://example.com/course/demo-course',
...
```

Reply:
done